src/lib/components/organisms/menu/menu.svelte

```diff
@@ -21,30 +20,63 @@
      */
     export let activeIndex: number | undefined = undefined
 
+    /**
+     * Is mobile menu open
+     */
+    export let isMobileMenuOpen: boolean = false
+
     function getActiveIndexSetter(index: number): () => void {
         return () => (activeIndex = index)
     }
+
+    function toggleMobileMenu(): void {
+        isMobileMenuOpen = !isMobileMenuOpen
+    }
 </script>
 
-<nav class="flex flex-row items-center justify-between w-full px-6 lg:px-0">
-    <Logo {darkmode} />
-    <desktop-menu-items class="hidden lg:flex flex-row items-center space-x-4">
-        {#each menuItems as menuItem, index}
-            {@const handleOnClick = getActiveIndexSetter(index)}
-            <MenuItem
-                {...menuItem}
-                {darkmode}
-                isActive={activeIndex === index}
-                on:click={handleOnClick}
-            />
-        {/each}
-    </desktop-menu-items>
-    <mobile-menu class="flex lg:hidden">
-        <Icon
-            icon={IconEnum.Menu}
-            width={32}
-            height={32}
-            fill={darkmode ? 'white' : CUSTOM_COLORS['iota-gray'][900]}
-        />
-    </mobile-menu>
+{#if isMobileMenuOpen}
+    <button
+        class="fixed z-[-1] top-full left-0 outline-none border-none p-0 w-full h-screen cursor-auto"
+        on:click={toggleMobileMenu}
+    />
+{/if}
+
+<nav class="container w-full" class:relative={isMobileMenuOpen}>
+    <div class="flex flex-row items-center justify-between px-6 py-3 md:py-0">
```

Comment:
I feel like there's a bit too much padding here.. :pray: 

![Screenshot from 2024-04-12 08-45-14](https://github.com/iotaledger/iota-ui-kit/assets/72078237/0279dd3c-9583-4bc2-ba80-9cc2d38d3eb0)
![Screenshot from 2024-04-12 08-48-35](https://github.com/iotaledger/iota-ui-kit/assets/72078237/3c93a7c5-e1d0-4023-9983-fe75d3fe66de)

Reply:
This is how the navbar is used. Maybe I could remove the container class and leave it to users to wrap the navbar with a container, but it will probably be needed in all the pages, that's why I decided to put it here. 
![image](https://github.com/iotaledger/iota-ui-kit/assets/88061365/64f25d03-fa33-4c31-840c-525372767960)
Same with the mobile one, it's 60px height.
![image](https://github.com/iotaledger/iota-ui-kit/assets/88061365/1a45e55b-f7f9-4f97-b581-22f31789a726)

Reply:
sure, no worries at all :blush: if that's the way it's supposed to be, then I'm totally cool with it :v: